src/content/docs/quick-tips/docker-image.mdx

```diff
@@ -75,7 +73,15 @@ Namely:
 - `/external/styles/` contains the user provided CSS Override
 - `/external/external/` is a placeholder directory that users can leverage to distribute data through Ontime 
 
-These three paths can be bound to individual host directories as in the sample docker-compose file, or you can bind a single host directory to /external/ for Ontime to store all its files in.
+You can bind a single host directory to `/external/` for Ontime to store all its files in, or
+these three paths can be bound to individual host directories as in the sample docker-compose file:
```

Comment:
suggestion, best not to mention the sample docker-compose file.
This could lead to confusion if the user doesnt know which file we mean or if we decide to edit the example in docker-hub

```suggestion
You can bind a single host directory to `/external/` for Ontime to store all its files in, or expose the individual parts to host directories:
```